### PR TITLE
Add editor config

### PR DESCRIPTION
**Pull request by bonustrack**
Just to make the code more readable on GitHub, before
https://github.com/balancer-labs/assets/blob/master/scripts/check-prices.js
After 
https://github.com/balancer-labs/assets/blob/8a8cca99fb8a6109f553f03faaa05cfb4b16ef0a/scripts/check-prices.js

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
```
Comment (Destiner) [inline .editorconfig:4]: To match the code style:
```suggestion
indent_style = tab
```

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+indent_size = 2
+end_of_line = LF
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
```
Comment (MGmarcineczek) [inline .editorconfig:11]: 0xf55041e37e12cd407ad00ce2910b8269b01263b9

**Review by MGmarcineczek — COMMENTED** [review]
0xA1dc5BDF51917B51E7Bd97BaD5546F27761669c0

File: .editorconfig (hunk 2 above)
Comment (MGmarcineczek) [reply .editorconfig:11]: 0xA1dc5BDF51917B51E7Bd97BaD5546F27761669c0

File: .editorconfig (hunk 1 above)
Comment (MGmarcineczek) [reply .editorconfig:4]: crypto/tools/README.md

**Comment by MGmarcineczek** [comment]
crypto/tools/README.md

**Pull request closed without merging**